src/main/scala/scala/tools/refactoring/implementations/OrganizeImports.scala

```diff
@@ -11,6 +11,40 @@ import transformation.TreeFactory
 import scala.collection.mutable.ListBuffer
 import scala.collection.mutable.LinkedHashMap
 import scala.util.control.NonFatal
+import scala.collection.immutable.Queue
+
+object OrganizeImports {
+  /**
+   * Abstract algorithms used by the implementation, extracted mostly for testing purposes
+   */
+  private[implementations] object Algos {
+    def groupImports[ImportT](getImportExpression: ImportT => String)(groups: Seq[String], imports: Seq[ImportT]): Seq[List[ImportT]] = {
```

Comment:
The type parameter is for making testing easier because you can test with strings instead of trees? :+1: